docs/management/admin/response-actions-config.asciidoc

```diff
@@ -80,6 +81,69 @@ IMPORTANT: Do not create more than one CrowdStrike connector.
 This gives you visibility into CrowdStrike without needing to leave {elastic-sec}. You can perform supported endpoint response actions directly from alerts that a rule creates, by using the **Take action** menu in the alert details flyout.
 ====
 
+.**Set up Microsoft Defender for Endpoint response actions**
+[%collapsible]
+====
+// NOTE TO CONTRIBUTORS: These sections have very similar content. If you change anything 
+// in this section, apply the change to the other sections, too.
+
+. **Create API access information in Microsoft Azure.** You should create two new applications in your Azure domain and grant them the following minimum API permissions:
++
+--
+- For use with the Microsoft Defender for Endpoint Fleet integration policy: Permission to read alert data (`Windows Defender ATP: Alert.Read.All`).
```

Comment:
This edit and the following one reflect the list structure used in step 1 of the "Set up SentinelOne response actions" section. 

```suggestion
- Microsoft Defender for Endpoint Fleet integration policy: Permission to read alert data (`Windows Defender ATP: Alert.Read.All`).
```